### DISCUSSION TITLE
Is Mongoose v6.9.0 compatible with MongoDB v5.x?

### DISCUSSION THREAD

**Discussion by susansiow** [Automattic/mongoose#12960] [Q&A] [answered]
I have received an email from MongoDB saying that MongoDB Atlas will be automatically upgraded to v6.0 in March 2023, therefore I would like to know if I can upgrade Mongoose `v6.2.1 to v6.9.0` installed in my backend before MongoDB v6.0 takes place?

1. Is Mongoose v6.9.0 compatible with MongoDB v5.x?
2. Are there any breaking changes to upgrade Mongoose from `v6.2.1 to v6.9.0`?

Kindly advise, thank you!

**Comment by vkarpov15** [comment, marked as answer]
1. Yes.
2. No intentional breaking changes. If you have issues, please open a new issue on GitHub

See https://mongoosejs.com/docs/compatibility.html for what versions of MongoDB Mongoose is compatible with.
